etc/behat/suites/ui-cart.yml

```diff
@@ -68,5 +68,6 @@ default:
                     taxCategoryRepository: "@sylius.repository.tax_category"
                     zoneRepository: "@sylius.repository.zone"
 
+                - Sylius\Behat\Context\Ui\UserContext
```

Comment:
Let's put `: ~` at the end to be sure that context class will be interpreted as a key.